src/it/resources/org/checkstyle/suppressionxpathfilter/booleanexpressioncomplexity/SuppressionXpathRegressionBooleanExpressionComplexityOne.java

```diff
@@ -0,0 +1,11 @@
+package org.checkstyle.suppressionxpathfilter.booleanexpressioncomplexity;
+
+public class SuppressionXpathRegressionBooleanExpressionComplexityOne {
+    public void methodOne() {
+        boolean a = true;
+        boolean b = false;
+
+        boolean c = (a & b) | (b ^ a); // OK
+        boolean d = (a & b) | (b ^ a) | (a & b) | (b ^ a); // warn
```

Comment:
Please make 3 test input files with violation at:
Class field, method variable , IF expression.

Generated xpath should be different by structure.

Reply:
Okay  @romani  sir !